modules/crop-ffmpeg/src/test/java/org/opencastproject/crop/CropServiceTest.java

```diff
@@ -0,0 +1,209 @@
+import org.opencastproject.serviceregistry.api.ServiceRegistryInMemoryImpl;
+import org.opencastproject.util.MimeTypes;
+import org.opencastproject.workspace.api.Workspace;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.tika.io.IOUtils;
+import org.easymock.EasyMock;
+import org.easymock.IAnswer;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.net.URI;
+
+public class CropServiceTest {
+
+  /** Video file to test. Has black bars on each side */
+  protected static final String mediaResource = "/black.mp4";
+
+  /** Video file to test entire black slides. Shoudn't not be cropped */
+  protected static final String mediaResource1 = "/schwarzesVideoFinal.mp4";
+
+  /** Duration of whole videos */
+  protected static final long mediaDuration = 40000L;
+  protected static final long mediaDuration1 = 6000L;
+
+  /** The in-memory service registration */
+  protected ServiceRegistry serviceRegistry = null;
+  protected ServiceRegistry serviceRegistry1 = null;
+
+  /** The corp services */
+  protected CropServiceImpl cropService = null;
+  protected CropServiceImpl cropService1 = null;
+
+  protected Mpeg7CatalogService mpeg7Service = null;
+  protected Mpeg7CatalogService mpeg7Service1 = null;
+
+  /** The media url */
+  protected static TrackImpl track = null;
+  protected static TrackImpl track1 = null;
+
+  protected File tempFile = null;
+  protected File tempFile1 = null;
+
+  @Rule
+  public TemporaryFolder testFolder = new TemporaryFolder();
+
+  /**
+   * Copies test files to the local file system.
+   *
+   * @throws Exception
+   *           if setup fails
+   */
+  @BeforeClass
+  public static void setUpClass() throws Exception {
+    track = TrackImpl.fromURI(CropServiceTest.class.getResource(mediaResource).toURI());
+    track.setFlavor(MediaPackageElements.PRESENTATION_SOURCE);
+    track.setMimeType(MimeTypes.MJPEG);
+    track.addStream(new VideoStreamImpl());
+    track.setDuration(mediaDuration);
+
+    track1 = TrackImpl.fromURI(CropServiceTest.class.getResource(mediaResource1).toURI());
+    track1.setFlavor(MediaPackageElements.PRESENTATION_SOURCE);
+    track1.setMimeType(MimeTypes.MJPEG);
+    track1.addStream(new VideoStreamImpl());
+    track1.setDuration(mediaDuration1);
+  }
+
+  @Before
+  public void setUp() throws Exception {
+    mpeg7Service = new Mpeg7CatalogService();
+    Workspace workspace = EasyMock.createNiceMock(Workspace.class);
+    EasyMock.expect(workspace.get((URI) EasyMock.anyObject())).andReturn(new File(track.getURI()));
+    tempFile = testFolder.newFile(getClass().getName() + ".xml");
+    EasyMock.expect(workspace.putInCollection((String) EasyMock.anyObject(), (String) EasyMock.anyObject(),
```

Comment:
You can make this simpler by using lambdas and the more specific easymock methods:

```diff
diff --git a/modules/crop-ffmpeg/src/test/java/org/opencastproject/crop/CropServiceTest.java b/modules/crop-ffmpeg/src/test/java/org/opencastproject/crop/CropServiceTest.java
index 6e78e464b..02d52f0e1 100644
--- a/modules/crop-ffmpeg/src/test/java/org/opencastproject/crop/CropServiceTest.java
+++ b/modules/crop-ffmpeg/src/test/java/org/opencastproject/crop/CropServiceTest.java
@@ -120,11 +120,8 @@ public class CropServiceTest {
     tempFile = testFolder.newFile(getClass().getName() + ".xml");
-    EasyMock.expect(workspace.putInCollection((String) EasyMock.anyObject(), (String) EasyMock.anyObject(),
-            (InputStream) EasyMock.anyObject())).andAnswer(new IAnswer<URI>() {
-      @Override
-      public URI answer() throws Throwable {
-        InputStream in = (InputStream) EasyMock.getCurrentArguments()[2];
-        IOUtils.copy(in, new FileOutputStream(tempFile));
-        return tempFile.toURI();
-      }
-    });
+    EasyMock.expect(workspace.putInCollection(EasyMock.anyString(), EasyMock.anyString(),
+            EasyMock.anyObject(InputStream.class))).andAnswer(() -> {
+              InputStream in = (InputStream) EasyMock.getCurrentArguments()[2];
+              IOUtils.copy(in, new FileOutputStream(tempFile));
+              return tempFile.toURI();
+            });
     EasyMock.replay(workspace);
```